auth-web/src/stores/business.ts

```diff
@@ -209,6 +209,21 @@ export const useBusinessStore = defineStore('business', () => {
     const entityResponse: AffiliationResponse[] = await OrgService.getAffiliatedEntities(currentOrganization.value.id)
     let affiliatedEntities: Business[] = []
 
+    // do a lear search on firms to fetch the latest names (alternate names array)
+    // set that array to firm entries in the entityResponse array
+    for (const response of entityResponse) {
```

Comment:
Basically, for firms, we need to do a LEAR search to grab that alternateNames array that we need. It's needed to properly display the latest correct name.

Reply:
This should be done in the API not on the frontend as a loop?

Reply:
Yeah, was thinking that as well. I'll be doing that.

Reply:
Yeah I remember we spent quite a bit of time getting rid of having to call Namex and Legal-api on a per business basis.. it caused alot of performance issues when loading up a dashboard with lets say 100 businesses

Reply:
As discussed, this https://github.com/bcgov/lear/blob/ce180c3abb996eefd5886def72ddc29620007925/legal-api/src/legal_api/models/business.py#L627

should populate the `/search` call

Reply:
Sounds good, on it

Reply:
Removed this entire block Travis, should be OK now. 

Search call is now grabbing the alternateNames array (I've made changes in LEAR): 
![search call](https://github.com/bcgov/sbc-auth/assets/122301442/b6a57bf8-2f50-40a2-b414-b0f112506de7)

MBR looks good:
![MBR looks good](https://github.com/bcgov/sbc-auth/assets/122301442/d3b2510a-00fa-4944-8753-5f40bb6ce348)